verification/src/http_server/routers/solidity.rs

```diff
@@ -29,8 +29,8 @@ impl Router for SolidityRouter {
     fn register_routes(&self, service_config: &mut web::ServiceConfig) {
         service_config.app_data(self.cache.clone()).service(
             web::scope("/verify")
-                .route("/flatten", web::post().to(flatten::verify))
-                .route("/standard_json", web::post().to(standard_json::verify)),
+                .route("/", web::post().to(files_input::verify))
```

Comment:
decided to leave this verification on `/api/v1/solidity/verify/`. We can move it to `/verify/files/` or something, but i don't think it's needed

Reply:
should I rename everything to "multi-part style". I mean rename structs to `MultiPart`, file to `multi-part.rs` and so on?